x-pack/solutions/security/plugins/security_solution/public/attack_discovery/pages/use_find_attack_discoveries/index.ts

```diff
@@ -50,10 +51,11 @@ interface PageParam {
   perPage?: number;
 }
 
-const DEFAULT_PAGE = 1; // CAUTION: sever-side API uses a 1-based page index convention (for consistency with similar existing APIs)
+const DEFAULT_PAGE = 1; // CAUTION: server-side API uses a 1-based page index convention (for consistency with similar existing APIs)
 const DEFAULT_PER_PAGE = 10;
 
 export const useFindAttackDiscoveries = ({
+  alertIds,
```

Comment:
This PR is not tagged as being backported to `8.19`, but consider backporting this change to simplify future backports and testing, (avoiding diverging implementations).

Reply:
Unfortunately, backporting this would lead to a cascade of backports including the following and likely more. I think we may be past that point... @PhilippeOberti what are your thoughts here?

https://github.com/elastic/kibana/pull/218018
https://github.com/elastic/kibana/pull/219004
https://github.com/elastic/kibana/pull/214889
https://github.com/elastic/kibana/pull/215246
https://github.com/elastic/kibana/pull/216744
https://github.com/elastic/kibana/pull/217421
https://github.com/elastic/kibana/pull/217696
https://github.com/elastic/kibana/pull/217744

Reply:
I don't think we should backport this... Like @stephmilovic said above, we'd have to backport a bunch of PRs ([24 to be exact](https://github.com/elastic/kibana/pulls?q=is%3Apr+author%3APhilippeOberti+is%3Aclosed+AI4DSOC+)) and I don't think it's a good idea...

Reply:
If we truly believe we should, I'll get started on backporting the PRs. Thankfully I kept most of them very targeted and with low impact on the rest of the code, but some of them will be more involved...

Reply:
The thing to keep in mind is many other PRs from other teams (outside of the 24 I linked above which are just for the alert summary work) have not been backported as well. So we'd have to sync with the rest of the group to make sure that we backport everything? Or you'd have a bunch of code in `8.19` that is untestable and it would make no sense... while also adding risk to break something in the process...

Reply:
Synced with Andrew over zoom. He just wants me to bring the changes to this specific file over to 8.19, since it exists there, in order to bring it up to speed with main. This is going to be a manual PR, not using the backport tool, so limited to the changes to this file. He wasn't suggesting we backport the whole PR, just the file changes. Sorry for the confusion @PhilippeOberti

Reply:
Haaaaaa that makes sense, all good then!! 😆